app/models/ability.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    return if user.blank?
+
+    if user.referencee.is_a? CivilServant
+      merge(Abilities::CivilServantAbility.new(user.referencee))
+    elsif user.referencee.is_a? OrganizationMember
```

Comment:
I purposely specified elsif to not mistakenly grant organisation member privileges on a possible new referencee